src/main/java/io/anserini/collection/RDFCollection.java

```diff
@@ -0,0 +1,131 @@
+package io.anserini.collection;
+
+import io.anserini.document.RDFDocument;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.zip.GZIPInputStream;
+
+/**
+ * Class representing an instance of an RDF collection.
+ * RDF datasets can represent knowledge bases such as Freebase.
+ * They often come in a single .gz file.
+ */
+public class RDFCollection extends Collection<RDFDocument> {
```

Comment:
By strict Java conventions, this should be RdfCollection, but I don't care that much...